quickjs.c

```diff
@@ -1774,10 +1774,25 @@ static int init_class_range(JSRuntime *rt, JSClassShortDef const *tab,
     return 0;
 }
 
-/* Note: OS and CPU dependent */
+/* Uses code from LLVM project. */
 static inline uintptr_t js_get_stack_pointer(void)
 {
+#ifdef _MSC_VER
+    return (uintptr_t)_AddressOfReturnAddress();
```

Comment:
You'll need to remove the code that defined `__builtin_frame_address` based on this, from cutils.{h,c}